db/migrate/20150622213333_add_person_id_to_tasks.rb

```diff
@@ -0,0 +1,5 @@
+class AddPersonIdToTasks < ActiveRecord::Migration
+  def change
+    add_column :tasks, :person_id, :FixNum
```

Comment:
This `:FixNum` as a column type in a Rails migration was new to me. Usually I would use `:integer` here. I tested your migrations and while this might work for sqlite it causes problems for the Rails migration, specifically the error message in your `db/schema.rb`, below. Changing it to `:integer` fixed the problem, but I had to drop the database and recreate it (so if you do this in development you'll lose all the data you've entered).

    rake db:drop db:create db:migrate

Also, I had a problem running the migrations from scratch and had to fix the first migration (create tasks) to change the class name to match what Rails expected from the filename.

```diff
-class CreateTaskTable < ActiveRecord::Migration
+class CreateTask < ActiveRecord::Migration
```